packages/components/src/notice/test/index.js

```diff
@@ -15,6 +15,10 @@ import Notice from '../index';
 
 jest.mock( '@wordpress/a11y', () => ( { speak: jest.fn() } ) );
 
+function getNoticeWrapper( container ) {
+	return container.firstChild;
+}
```

Comment:
Creating a wrapper function for this doesn't only allow us to reduce code repetition and have the violation only in one place; it actually tricks the linter - there's no need for a rule ignore here.